Replicator/IncomingRev.cc

```diff
@@ -264,6 +264,8 @@ namespace litecore { namespace repl {
         // SG sends a fake revision with a "_removed":true property, to indicate that the doc is
         // no longer accessible (not in any channel the client has access to.)
         if (root["_removed"_sl].asBool()) {
+            logInfo("SG removing: %.*s.%.*s.%.*s/%.*s", SPLAT(_rev->collectionSpec.scope), SPLAT(_rev->collectionSpec.name),
+                    SPLAT(_rev->docID), SPLAT(_rev->revID));
```

Comment:
"Received removed revision for doc : scope.collection.docid/revid"

Reply:
okay

Reply:
Can you change the removed doc to the removed revision? It's the term that we are using for this special rev.

Reply:
yes

Reply:
Actually, this is a fake revision, 
        // SG sends a fake revision with a "_removed":true property, to indicate that the doc is
        // no longer accessible (not in any channel the client has access to.)
Revision is not important. Actually, the doc will be purged, not the revison.

Reply:
revision here is only a technical thing. We log it here to help debugging.

Reply:
What we received is the revision from getRev so technically it's the rev. You can also be specific like "Received the fake removed revision ..."

Reply:
OK, I'm fine either way actually.